agent/config/config_test.go

```diff
@@ -592,6 +601,49 @@ func TestUserDataConfig(t *testing.T) {
 	}
 }
 
```

Comment:
Add test cases for the key:value pairs?

Reply:
I think I've already added [here](https://github.com/aws/amazon-ecs-agent/pull/1618/commits/ed4f271d761ebb56d6c1836e1574ff93a7dfeaae#diff-540a525277ce057e5db7c092fb41f78fR90). LMK if you want me to include more test cases.